waku/node/v2/api/waku_api_rpc.nim

```diff
@@ -0,0 +1,23 @@
+import
+  json_rpc/rpcserver,
+  eth/async_utils,
+  waku_api_types, waku_api_tools,
+  ../waku_types,../wakunode2
+
+proc setupWakuRPCAPI*(node: WakuNode, rpcsrv: RpcServer) =
+  
+  rpcsrv.rpc("get_waku_v2_history") do(query: HistoryQueryAPI) -> HistoryResponseAPI:
```

Comment:
```suggestion
  rpcsrv.rpc("get_waku_v2_history") do(query: HistoryQueryAPI) -> HistoryResponseAPI:
```

I really hate this name, will this be what the function is called or how exactly will this work?

Reply:
This is done to be explicit with mapping to REST API, see https://github.com/status-im/nim-waku/issues/243 This means it corresponds to a GET call with `/waku/v2/history` endpoint. How would you solve mapping problem without it?

My only comment here would be that it might make sense to call it `store` instead of `history` to make mapping more explicit. On the other hand, store is more of a verb, and history is the resource, so maybe it does make sense from a REST POV. @jm-clius do you have any thoughts on what the other endpoints would be called and whether Store API or History API makes more sense?

Reply:
Mmm. After giving this some thought I'm backtracking on using `history`.  I used "get...history" because it felt more natural for a read action than "get...store", but this unnecessarily diverges from the underlying protocols. Endpoints for `relay` and `filter` will be called just that, so I'd rather maintain consistency.
The proposed scheme for naming read methods are `get_waku_v2_<API>_<resource>`. The most explicit mapping for this method would then be `get_waku_v2_store_query`.